app/views/navigation_rules/edit.html.erb

```diff
@@ -25,6 +25,8 @@
   action: 'Choose on-page side navigation'
 %>
 
+<%= render 'step_by_step_pages/nav', step_by_step_page: @step_by_step_page, active:"choose-navigation" %>
```

Comment:
Nitpick that @andysellick originally spotted: There should be a space between `:` and `"`